### DISCUSSION TITLE
Similar diffusion based paper just got released

### DISCUSSION THREAD

**Discussion by francqz31** [lucidrains/meshgpt-pytorch#29] [Ideas]
There is a similar diffusion based paper that just got released.              PS: I think it is worse than Meshgpt

https://arxiv.org/abs/2312.11417

That might be the start for something

**Comment by lucidrains** [comment]
It is the same research group

they are planting the flag for both major generative approaches

**Reply by francqz31** [reply]
Yeah I know right , not just this, The same approach and idea of meshgpt and the other can be used for molecular generation!!

**Reply by lucidrains** [reply]
yeah it should be tried, although, I think relative distances of the vertices (atoms) will still be important and needs to be injected into the GNN

**Reply by lucidrains** [reply]
but I have been mulling it over how to adapt this to bond generation (also questioning whether it will have any benefits). may execute once it is clear in my head

**Reply by lucidrains** [reply]
are you a researcher?

**Reply by francqz31** [reply]
not really but I'm in the making I guess :)

**Comment by MarcusLoppe** [comment]
> There is a similar diffusion based paper that just got released. PS: I think it is worse than Meshgpt
> 
> https://arxiv.org/abs/2312.11417
> 
> That might be the start for something

I tried to compare the accuracy to MeshGPT but they use different  test results in the papers, both papers use the same test dataset and include models: AtlasNet ,Polygen and BSPNet.
But they made an error with the test so the results are inconsistent and not comparable.

![bild](https://github.com/lucidrains/meshgpt-pytorch/assets/65302107/0e641882-37fb-47ff-b0f3-e5b834fcaa17)



@lucidrains It seems like they have the same issue regarding the faces, both their paper limits the faces to 800. 
I'm guessing they are running into VRAM issues as well, or maybe out of the training budget.
